src/main/java/Task.java

```diff
@@ -8,12 +8,14 @@ public Task(String description) {
     }
 
     public String getStatusIcon() {
-        return "[" +(isDone ? "\u2713" : "\u2718")+"]" ; //return tick or X symbols
+        //return tick or X symbols
```

Comment:
I like how you add comments on for unicode.

Reply:
yes, as I might not know what unicode represents after few weeks. 👍